_config.yml

```diff
@@ -16,10 +16,18 @@
 title: Just the Docs
 description: A Jekyll theme for documentation
 baseurl: "/just-the-docs" # the subpath of your site, e.g. /blog
-url: "https://pmarsceill.github.io" # the base hostname & protocol for your site, e.g. http://example.com
+url: "https://pdmosses.github.io" # the base hostname & protocol for your site, e.g. http://example.com
 
 permalink: pretty
-exclude: ["node_modules/", "*.gemspec", "*.gem", "Gemfile", "Gemfile.lock", "package.json", "package-lock.json",  "script/", "LICENSE.txt", "lib/", "bin/", "README.md", "Rakefile"]
+exclude: ["node_modules/", "*.gemspec", "*.gem", "Gemfile", "Gemfile.lock", "package.json", "package-lock.json",  "script/", "LICENSE.txt", "lib/", "bin/", "README.md", "Rakefile"
+, "docs/tests/"
+]
+
+# Regression tests
```

Comment:
I love this idea... A note for the future:

We could create a separate `_config.yml` file that includes the tests and then y[ou can use that instead of the default if you wanted to run the tests](https://jekyllrb.com/docs/usage/).

```
jekyll serve --config _regression_tests.yml
```

Then we wouldn't need to comment anything out, they would just live in a different file.

Reply:
Yes, that is indeed a much neater way of activating the tests locally. Next release I guess.